REFERENCE_TABLES.py

```diff
@@ -0,0 +1,8 @@
+from astrodbkit2.astrodb import create_database,Database
+from simple.schema import *
+
+connection_string = 'sqlite:///SIMPLE.db.py' # connection string for a SQLite database named SIMPLE.db.py
+create_database(connection_string)
+db = Database(connection_string)
+
+db.load_database('data/')
```

Comment:
Not sure where you got this from. Please use your `filip_15.py` script as an example and use the `load_simpledb` function.

Reply:
I deleted that file, honestly no clue where it came from